signing/secp256k1/api/src/main/java/tech/pegasys/signers/secp256k1/api/SignerProvider.java

```diff
@@ -20,6 +20,11 @@
 
   Optional<Signer> getSigner(ECPublicKey publicKey);
 
+  default Optional<Signer> getSigner(String address) {
```

Comment:
Is address always an ethereum address? How will this work for filecoin addresses? Is this just the string representation of a filecoin address?